vhdbuilder/packer/generate-windows-vhd-configuration.ps1

```diff
@@ -20,10 +20,9 @@ if (-not ($validSKU -contains $windowsSKU)) {
 # You must install the April 22, 2021 servicing stack update (SSU) (KB5001407) before installing the latest cumulative update (LCU).
 # SSUs improve the reliability of the update process to mitigate potential issues while installing the LCU. 
 $global:patchUrls = @(
-    "http://download.windowsupdate.com/c/msdownload/update/software/secu/2021/05/windows10.0-kb5003243-x64_81350c4efec5a183725fda73091c9ee9d4577bc3.msu",
-    "http://download.windowsupdate.com/c/msdownload/update/software/secu/2021/05/windows10.0-kb5003171-x64_30162051d5376b7a19c4c25157347c522e804bbb.msu"
+    "http://download.windowsupdate.com/d/msdownload/update/software/updt/2021/05/windows10.0-kb5003217-x64_92c969426f41f135f842dd232a64e6a461a848ca.msu"
 )
-$global:patchIDs = @("KB5003243", "KB5003171")
+$global:patchIDs = @("kb5003217")
```

Comment:
Let us to check the check-in result to see whether we still need KB5003243. https://support.microsoft.com/en-us/topic/may-20-2021-kb5003217-os-build-17763-1971-preview-08687c95-0740-421b-a205-54aa2c716b46

Before installing this update
```
Prerequisite:

You must install the May 11, 2021 servicing stack update (SSU) (KB5003243) or later before installing the latest cumulative update (LCU). 
```

Reply:
The release note says KB5003243  is installed
Installed QFEs
	KB4601060 : Update          : http://support.microsoft.com/?kbid=4601060
	KB5003243 : Security Update : http://support.microsoft.com/?kbid=5003243
	KB5003217 : Update          : http://support.microsoft.com/?kbid=5003217